tests/test_connections_page.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from utilities.data_randomizer import generate_invalid_invitation_code
+
+
+@pytest.mark.smoke
+@pytest.mark.regression
+def test_update_connection_status(go_to_connections_page):
+    connections_page = go_to_connections_page
+    # activate > deactivated
+    active_status = connections_page.get_connection_status()
```

Comment:
it is good to verify that activate status is correct at this point

Reply:
added extended condition, so the update can be executed despite the initial status.